<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HokusGuardMainWindow</class>
 <widget class="QMainWindow" name="HokusGuardMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HokusGuard - the better guard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="general.qrc">
    <normaloff>:/images/potter.svg</normaloff>:/images/potter.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
	background-color:#fa2569;
}

QMessageBox 
{
    background-color: #220c28;
}

QMessageBox QLabel 
{
    	background-color:rgba(0, 0, 0,0);
	border: 1px solid grey;
	border-color:#220c28;
	border-radius: 5px;
	color:white;
}

QMessageBox QPushButton
{
	border-radius:15px;
	border: 1px solid grey;
	border-color:#fa2569;
	background-color:#220c28;
	color:white;
}

QMessageBox QPushButton::hover
{
	border-radius:15px;
	border: 1px solid grey;
	border-color:#feb83d;
	background-color:#220c28;
	color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border-radius:15px;
	background-color:#fa2569;
	color:white;
}

QPushButton:hover
{
	border-radius:15px;
	background-color:#feb83d;
	color:black;
}

QListWidget
{
	background-color:rgba(0, 0, 0,0);
	border: 1px solid grey;
	border-color:#feb83d;
	border-radius: 5px;
	color:white;
}

#mainframe
{
	background-color:#220c28;
	border-radius:10px;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="startGuard_pushButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>530</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Monitoring</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addMailPass_pushButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="general.qrc">
       <normaloff>:/images/plus.svg</normaloff>:/images/plus.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteMailPass_pushButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="general.qrc">
       <normaloff>:/images/minus.svg</normaloff>:/images/minus.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stopGuard_pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>530</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Monitoring</string>
     </property>
    </widget>
    <widget class="QListWidget" name="mail_listWidget">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>531</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveMail_pushButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>SAVE MAIL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteMail_pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>DEL MAIL</string>
     </property>
    </widget>
    <widget class="QListWidget" name="mailpass_listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>201</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="settings_pushButton">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="general.qrc">
       <normaloff>:/images/settings.svg</normaloff>:/images/settings.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="general.qrc"/>
 </resources>
 <connections/>
</ui>
